versioned_docs/version-6.x/ecosystem-vite.md

```diff
@@ -53,25 +53,122 @@ single-spa works well with native modules, systemjs, or even both. With Vite + s
 
-## vite-plugin-single-spa
+## vite-plugin-single-spa (v0.7.0)
 
-This is a new entry that is currently in the early stages of development, but shows significant progress ([view in GitHub](https://github.com/WJSoftware/vite-plugin-single-spa)). It claims to be able to convert out-of-the-box Vite projects (regardless of the framework) into single-spa micro-frontend projects and even root config projects. While the single-spa team discourages the use of UI frameworks in root configs, it is indeed an alternative that may interest people.
+This is a relatively new plug-in (1 year old as of May, 2024) capable of producing a bundle that conforms to single-spa's specifications for micro-frontend projects that are bundled with Vite.
 
-To convert a Vite project to a root config project, all that is needed is install `vite-plugin-single-spa`, and then use it in `vite.config.ts`. This is a Vite + Vue example:
+In a nutshell, any *Vite + [framework of choice]* project created with `npm create vite` can take advantage of this plug-in.  Configuration is usually minimal, and the resulting project will be capable of working as a single-spa micro-frontend and as a standalone project simultaneously (while in *serve* mode).
+
+The plug-in can also be used in root config projects that are bundled wth Vite.
+
+### Main Features
+
+For micro-frontends:
+
+- Supports stock Vite projects, regardless of framework.
+- Micro-frontend projects behave dually while in *serve* mode: The micro-frontend can be previewed as a standalone web application with its server URL, or it can be served as a single-spa micro-frontend.
+- Provides mounting and unmounting of CSS.  The algorithm:
+    + Supports Vite's code splitting
+    + Supports parcels
+    + Supports mounting the same micro-frontend or parcel multiple times (**read the documentation**)
+    + Provides FOUC (flash of unstyled content) prevention
+
+For root config projects:
+
+- Automatically picks up import maps from `src/importMap.dev.json` and `src/importMap.json`.
+- Automatically adds the `import-map-overrides` NPM package, user interface included.
+- Can merge multiple import map JSON files.
+
+:::info
+
+This plug-in (or better stated, *Vite*), will produce bundles in ES module format by default.  The recommendation when using this plug-in is to **keep it this way**.  However, feel free to add rollup configuration options to request the output in the SystemJS module format if you wish.  Just keep in mind that all of the plug-in's documentation assumes native ES modules.
```

Comment:
Can we mention it only works with native modules? Mixing native modules in development (since esbuild doesn't support System), and System can lead to problems.

Reply:
Updated.

Reply:
There is actually a way to use HMR with React by using interop script: https://github.com/MilanKovacic/single-spa-vite/blob/main/root-config/index.html#L11

But that can be added in the future.